ufc/bandit-flags-v1.json

```diff
@@ -286,7 +311,7 @@
                     "variationValue": "banner_bandit"
                 }
             ],
-            "modelVersion": "v123"
+            "modelVersion": "123"
```

Comment:
Removing the `v` to mirror the production change we're making. It doesn't make a material difference as long as this is consistent with `bandit-models-v1.json`